Icon titles

uruvelakassapa opened the pull request:
Added icon titles for desktop and mobile respectively to further clarify functionality available
<img width="752" alt="image" src="https://github.com/user-attachments/assets/c007a2c2-2634-4c78-8826-e5442a955302" />
<img width="511" alt="image" src="https://github.com/user-attachments/assets/8c0f171b-86f7-405d-aab9-d2ec8415c13a" />

Reptilioo commented:
It looks great.
There is just an issue with the button leading to `/advanced-search.html` at the right of the search bar, the padding got applied to it, as well as the hovering animation. 
Removing `.icon-button` on this element would remove the tooltip displaying when hovering the icon, but adding `.advanced-icon icon-button{padding: 0;}` would work. 
Removing `margin-top: 3px;` from `.advanced-icon` would also fix the vertical centering that is off.

Also, did you remove the tooltips added yesterday on the `github`, `youtube` and `support us` on purpose? I believe these 3 could still be of use.

Reptilioo commented:
@uruvelakassapa Is this a left-over that is not intended to be used or did you also want to add the padding and hovering animation on the 3 top left buttons? 
https://github.com/bkhpanigha/hh-suttas/blob/137131a70edf902f3140b0bc5421cca3595776ff/index.css#L1015-L1023

uruvelakassapa commented:
> @uruvelakassapa Is this a left-over that is not intended to be used or did you also want to add the padding and hovering animation on the 3 top left buttons?
> 
> https://github.com/bkhpanigha/hh-suttas/blob/137131a70edf902f3140b0bc5421cca3595776ff/index.css#L1015-L1023

I'm not on my laptop anymore but I believe the hover functionality was already there? In any case I'm happy to remove it as it doesn't add much to the experience.

Reptilioo commented:
@uruvelakassapa Yes, there was already an hover animation on the top left buttons that increased the opacity of the icons.
In this PR you added another hover animation on the buttons, which makes the background of the button gray, but this new animation was also added to the top left buttons, not just the top center buttons.

I added the changes so the top left buttons keep their previous hover animation, while the top center buttons get the new one.
Hopefully that was what you intended, otherwise let me know.

uruvelakassapa commented:
> @uruvelakassapa Yes, there was already an hover animation on the top left buttons that increased the opacity of the icons. In this PR you added another hover animation on the buttons, which makes the background of the button gray, but this new animation was also added to the top left buttons, not just the top center buttons.
> 
> I added the changes so the top left buttons keep their previous hover animation, while the top center buttons get the new one. Hopefully that was what you intended, otherwise let me know.

I see! Yeo that was not intended. Thanks for catching that.

Reptilioo commented:
@bkhpanigha If you approve these changes, it is ready to be merged.
